web/stripe-template/api/routes/GET-subscription-callback.ts

```diff
@@ -1,33 +1,34 @@
 import { RouteContext } from "gadget-server";
 import { stripe } from "../stripe";
 
-/**
- * Route handler for GET subscription-callback
- *
- * @param { RouteContext } route context - see: https://docs.gadget.dev/guides/http-routes/route-configuration#route-context
- */
 export default async function route({
   request,
   reply,
   api,
   logger,
   connections,
   currentAppUrl,
-}) {
-  const { query } = request;
+}: RouteContext) {
+  const { session_id, user_id } = request.query as {
+    session_id: string;
+    user_id: string;
+  };
 
-  const checkoutSession = await stripe.checkout.sessions.retrieve(
-    query.session_id
-  );
+  const checkoutSession = await stripe.checkout.sessions.retrieve(session_id);
 
-  logger.info({ checkoutSession }, "checkoutSession");
+  const stripeCustomerId = checkoutSession.customer as string;
```

Comment:
I think we can add a null check for `checkoutSession.customer` before this. Keep the string cast though